backend/controller/identity/dal/internal/sql/queries.sql

```diff
@@ -0,0 +1,8 @@
+-- name: GetOnlyIdentityKey :one
+SELECT private, public, verify_signature
+FROM identity_keys
+WHERE id = 1;
```

Comment:
It's not a good idea to rely on the id being 1, as weird stuff can happen with sequences, and we don't know what might happen in the future. I'd just remove the WHERE and if we return more than 1 it will fail.

Reply:
Changed for this PR because migration. Other changes in the next PR.